src/ofxSpacebrew.cpp

```diff
@@ -398,6 +398,7 @@ namespace Spacebrew {
 		if ( bConnected ){
         #ifdef SPACEBREW_USE_OFX_LWS
             client.send( m.getJSON( config.clientName ) );
+            ofNotifyEvent(onSendEvent, m.name, this);
```

Comment:
I think this is the only thing I would change!
1. It should be outside of the SPACEBREW_USE_OFX_LWS preprocessor block
2. I think it should pass the Message. Technically, apps can have > 1 publisher with the same name as long as they are different types!